Dockerfile

```diff
@@ -1,5 +1,8 @@
 FROM photon:3.0
 
+ARG KCTRL_VER=development
+LABEL kapp_controller_version=$KCTRL_VER
```

Comment:
any reason to label this image? (if we really want to, then lets namespace label `kapp-controller.carvel.dev/version=...`)

Reply:
thought it would be thorough somehow, but say no more! it's gone